extensions_admin/pulp_docker/extensions/admin/manifest.py

```diff
@@ -0,0 +1,118 @@
+from gettext import gettext as _
+
+from pulp.client.commands import options
+from pulp.client.commands.criteria import DisplayUnitAssociationsCommand
+from pulp.client.commands.unit import UnitCopyCommand, UnitRemoveCommand
+
+from pulp_docker.common.models import Manifest, Blob
+
+DESC_COPY = _('copies manifests from one repository into another')
+DESC_REMOVE = _('remove manifests from a repository')
+DESC_SEARCH = _('search for manifests in a repository')
+FORMAT_ERR = _('The docker manifest formatter can not process %s units.')
+
+UNIT_ID_TEMPLATE = '%(digest)s'
```

Comment:
Oh, man .. I love this!  Do you know which version of python 2 this was introduced in?